python/tvm/relay/backend/contrib/ethosu/te/unary_elementwise.py

```diff
@@ -111,7 +112,11 @@ def unary_elementwise_compute(
         "rounding_mode": rounding_mode,
     }
 
-    operators = {"ABS": te.abs}
+    def clz_imp(inp):
+        # Assuming that it's a 32 bit int
+        return 32 - te.log2(inp)
```

Comment:
Can ```inp``` be negative?

Reply:
Yes, the `inp` can be negative, but for the operators we turn into `extern_calls` the actual correctness of the compute expression doesn't really matter since we will pass to the NPU the parameters of the operation, but no specifics about the actual compute. That's probably why a "close enough" operation is used here...

Reply:
Yes, you are right.